lib/include/elements/support/point.hpp

```diff
@@ -24,6 +24,9 @@ namespace cycfi { namespace elements
       constexpr point   move(float dx, float dy) const;
       constexpr point   move_to(float x, float y) const;
 
+      constexpr float   coord(bool is_x_axis) const;
```

Comment:
Would an operator [] be useful here as well?
`constexpr float operator[](bool is_x_axis) const`